src/Analysis/GWASV2/Components/SelectConfiguration/SelectConfiguration.test.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import SelectConfiguration from './SelectConfiguration';
+import ValidInitialState from '../../TestData/InitialStates/ValidInitialState';
+import ACTIONS from '../../Utils/StateManagement/Actions';
+
+describe('SelectConfiguration component', () => {
+  let wrapper;
+  const dispatch = jest.fn();
+  const {
+    numOfPCs,
+    mafThreshold,
+    selectedStudyPopulationCohort,
+    covariates,
+    outcome,
+    imputationScore,
+  } = ValidInitialState;
+
+  beforeEach(() => {
+    wrapper = shallow(
+      <SelectConfiguration
+        numOfPCs={numOfPCs}
+        mafThreshold={mafThreshold}
+        selectedCohort={selectedStudyPopulationCohort}
+        covariates={covariates}
+        outcome={outcome}
+        dispatch={dispatch}
+        imputationScore={imputationScore}
+      />,
+    );
+  });
+
+  it('should render the SelectHareDropDown component', () => {
+    expect(wrapper.find('#input-selectHareDropDown').exists()).toBe(true);
```

Comment:
this one is actually an interesting one to test since it should call dispatch multiple times (set hare & population size)

Reply:
It calls
```js
    dispatch({
      type: ACTIONS.UPDATE_SELECTED_HARE,
      payload: selectedHareOutput,
    });
    dispatch({
      type: ACTIONS.UPDATE_FINAL_POPULATION_SIZES,
      payload: getFinalPopulationSizes(h),
    });
```